### ISSUE TITLE
Using module in MEAN.JS

### ISSUE THREAD

**Issue by staminna** [mjhea0/braintree-paywall#1]
@Hi mjhea0,

I am trying to make it work in a MEAN.JS application context but I am having trouble registering the routes. 

I added `var paywall = require('braintree-paywall');` into my config/express.js file

then of course the /checkout and all the other routes path always hit 404.
I needed to create a server.js file in the project root folder, and it's still empty. Perhaps moving the bin/www file to server.js and adapt app accordingly?

Here is the project root folder:
![screenshot 2017-10-23 18 52 29](https://user-images.githubusercontent.com/515180/31904649-c5afa598-b823-11e7-9d89-20b8c82ce63b.png)

Can you perhaps point out how it could possibly work (or not)?
 
Thanks

**Comment by mjhea0** [comment]
This project is a full-blown example. It's not a module, so you cannot just require it. You will have to move the routes and logic into your app.

**Comment by staminna** [comment]
I adapted the code for require to load it and it's running, but there is always an error in processing the payments even if the fields aren't red. I tested with my own cards. Shouldn't it work in sandbox mode? 
The tests run well though, and I though it could have been something I did wrong, so I cloned the repository again, without any modifications ran the instructions again with the keys but the processing error remained the same. Perhaps something changed in the API?
